bucket/neovide.json

```diff
@@ -0,0 +1,18 @@
+{
+  "homepage": "https://github.com/Kethku/neovide",
+  "description": "No Nonsense Neovim Client in Rust",
+  "version": "0.5.0",
+  "license": {
+    "identifier": "MIT",
+    "url": "https://github.com/Kethku/neovide/blob/master/LICENSE"
+  },
+  "url": "https://github.com/Kethku/neovide/releases/download/0.5.0/neovide.exe",
+  "hash": "FA8453D74B6833E7912C049FAA9880544CB629E3E610F1717639C8198A26CD85",
+  "bin": "neovide.exe",
+  "checkver": {
+    "github": "https://github.com/Kethku/neovide"
+  },
+  "autoupdate": {
+    "url": "https://github.com/Kethku/neovide/releases/download/$version/neovide.exe"
+  }
```

Comment:
```suggestion
{
    "version": "0.5.0",
    "description": "A simple GUI for Neovim",
    "homepage": "https://github.com/Kethku/neovide",
    "license": "MIT",
    "suggest": {
        "neovim": "neovim"
    },
    "architecture": {
        "64bit": {
            "url": "https://github.com/Kethku/neovide/releases/download/0.5.0/neovide.exe",
            "hash": "fa8453d74b6833e7912c049faa9880544cb629e3e610f1717639c8198a26cd85"
        }
    },
    "bin": "neovide.exe",
    "shortcuts": [
        [
            "neovide.exe",
            "Neovide"
        ]
    ],
    "checkver": "github",
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/Kethku/neovide/releases/download/$version/neovide.exe"
            }
        }
    }
}
```

Reply:
Hey, sorry this was my first app manifest and I didn't realize this neovide needed neovim.
I'll update it.

Doesn't `checkver` require github url?

Reply:
https://scoop-docs.now.sh/docs/concepts/App-Manifest-Autoupdate.html#adding-checkver-to-a-manifest

Reply:
https://scoop-docs.now.sh/docs/concepts/App-Manifest-Autoupdate.html#adding-checkver-to-a-manifest